cpu/solver/cgs_kernels.cpp

```diff
@@ -50,8 +50,6 @@ void initialize(std::shared_ptr<const DefaultExecutor> exec,
                 matrix::Dense<ValueType> *q, matrix::Dense<ValueType> *u,
                 matrix::Dense<ValueType> *u_hat,
                 matrix::Dense<ValueType> *v_hat, matrix::Dense<ValueType> *t,
-                matrix::Dense<ValueType> *alpha, matrix::Dense<ValueType> *beta,
-                matrix::Dense<ValueType> *gamma,
```

Comment:
I think you have some problem here. @gflegar put these back in 62ef76ce. It looks like you have merge/rebased with develop but did not favor the changes from develop.